packages/json/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.5.0)
 project(soss-json)
 
 find_package(soss-core REQUIRED)
+find_package(yaml-cpp)
```

Comment:
I'm guessing this `find_package(~)` call was necessary due to differences in how `yaml-cpp` is packaged on Windows?

Since `yaml-cpp` is a public dependency of `soss-core`, we should probably add

```
include(CMakeFindDependencyMacro)
find_dependency(yaml-cpp)
```

into [soss-coreConfig.cmake](https://github.com/osrf/soss_v2/blob/master/packages/core/soss-coreConfig.cmake). Then we wouldn't need to have this here, or in any other downstream plugins.